### PR TITLE
Pres exchange context

### PR DESCRIPTION
**Pull request by brentzundel**
Adds language to the presentation exchange context and describes what is needed for the `format` property

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: spec/spec.md [hunk 1]
```diff
@@ -154,16 +154,46 @@ was presented separately at IIW and almost immediately became a cornerstone of
...
-credential. This signature allows for a zero knowledge proof of the original 
-signature, allowing the prover to derive a `BbsBlsSignatureProof2020` that will 
-verify the disclosed claims. 
+A `presentation definition` that complies with this specification MUST contain a
+`format` property. The value of the `format` property MUST be an object with the
+following property:
+- `ldp_vp`: This property indicates that a W3C Verifiable Presentation will be
+submitted in the form of a JSON object. The value of this property MUST be an
+object with a `proof_type` property that has a value of either
+`BbsBlsSignatureProof2020` or `BbsBlsBoundSignatureProof2020`.
```
Comment (OR13) [inline spec/spec.md:171]:
```suggestion
`BbsBlsSignature2020`, `JsonWebSignature2020`, or `Ed25519Signature2018`.
```

Comment (OR13) [reply spec/spec.md:171]: the VP does not need to be related to the `derivedCredential` that is in the `verifiableCredential` property of the VP.

File: spec/spec.md [hunk 2]
```diff
@@ -154,16 +154,46 @@ was presented separately at IIW and almost immediately became a cornerstone of
...
 
-More information about how this works can be found in the [Linked Data Proof BBS+ Signatures 2020 Suite](https://w3c-ccg.github.io/ldp-bbs2020/#the-bbs-signature-proof-suite-2020) and the [Mattr example implementation](https://github.com/mattrglobal/jsonld-signatures-bbs). For a more general overview of LD-Framing strategies as a general approach to querying and matching Linked-Data, see the [JSON-LD Framing](https://json-ld.org/spec/FCGS/json-ld-framing/20180607/#framing) guide written by the JSON-LD Community Group at W3C on the occasion of version 1.1 of the JSON-LD specification.
+```json5
+{
+  "presentation_definition": {
+    "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+    "input_descriptors": [],
+    "format": {
+      "ldp_vp": {
+        "proof_type": ["BbsBlsSignatureProof2020", "BbsBlsBoundSignatureProof2020"]
```
Comment (OR13) [inline spec/spec.md:182]:
```suggestion
        "proof_type": ["BbsBlsSignature2020", "JsonWebSignature2020", "Ed25519Signature2018"]
```

**Review by OR13 — CHANGES_REQUESTED** [review]
I think we don't are the suite of the presentation.

We DO care that the credential type is `BbsBlsSignatureProof2020`.

**Comment by OR13** [comment]
There exist merge conflicts.

**Comment by brentzundel** [comment]
@OR13 @TelegramSam merge conflicts resolved

**Review by OR13 — APPROVED** [review]

**Pull request merged**